src/tools/index.ts

```diff
@@ -152,4 +154,104 @@ export function shopifyTools(server: McpServer) {
+
+      Args:
+      paths: The paths to the documents to read, in a comma separated list.
+      Paths should be relative to the root of the developer documentation site.`,
+      {
+        paths: z
+          .array(z.string())
+          .describe("The paths to the documents to read"),
+      },
+      async ({ paths }) => {
+        async function fetchDocText(path: string): Promise<{
+          text: string;
+          path: string;
+          success: boolean;
+        }> {
+          try {
+            const appendedPath = path.endsWith(".txt") ? path : `${path}.txt`;
+            const url = new URL(appendedPath, SHOPIFY_BASE_URL);
+            const response = await fetch(url.toString());
+            const text = await response.text();
+            return { text: `## ${path}\n\n${text}\n\n`, path, success: true };
+          } catch (error) {
+            return {
+              text: `Error fetching document at ${path}: ${error}`,
+              path,
+              success: false,
+            };
+          }
+        }
+
+        const fetchDocs = paths.map((path) => fetchDocText(path));
+        const results = await Promise.all(fetchDocs);
+
+        return {
+          content: [{ type: "text" as const, text: results.join("---\n\n") }],
+        };
+      },
+    );
+
+    const surfaces = [
+      "app-home",
+      "admin-extensions",
+      "checkout-extensions",
+      "customer-account-extensions",
+    ] as const;
+    server.tool(
+      "get_started",
+      `
+      1. Ask user for the surface they are building for.
+      2. Use read_docs tool to read the docs for the surface.
+
+      Whenever the user asks about Polaris web components, always use this tool first to provide the most accurate and up-to-date documentation.
+
+      valid arguments for this tool are:
+      - "app-home"
+      - "admin-extensions"
+      - "checkout-extensions"
+      - "customer-account-extensions"
+
+      Once you determine the surface, you should then use the read_docs tool to learn about more specific details. Overviews are not comprehensive, so this is important.
+
+      DON'T SEARCH THE WEB WHEN REFERENCING INFORMATION FROM THIS DOCUMENTATION. IT WILL NOT BE ACCURATE. ONLY USE THE read_docs TOOLS TO RETRIEVE INFORMATION FROM THE DEVELOPER DOCUMENTATION SITE.
+    `,
+      {
+        surface: z
+          .enum(surfaces)
+          .describe("The Shopify surface you are building for"),
+      },
+      async function cb({ surface }) {
+        if (!surfaces.includes(surface)) {
+          const options = surfaces.map((s) => `- ${s}`).join("\n");
+          const text = `Please specify which Shopify surface you are building for. Valid options are: ${options}.`;
+
+          return {
+            content: [{ type: "text" as const, text }],
+          };
+        }
+
+        const docEntrypointsBySurface: Record<string, string> = {
+          "app-home": "/docs/api/app-home/using-polaris-components",
```

Comment:
Is there a reason we are using `using-polaris-components` instead of just `docs/api/app-home`
```suggestion
          "app-home": "/docs/api/app-home",
```

Reply:
I was conflicted about that, but I went with `/api/app-home/using-polaris-components` because `/api/app-home` doesn't actually have any `s-*` text on the page. 

That being said, I have no objection to this suggestion. 

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/f0ae3c6a-9cde-4dad-a526-791d49722936" />

## References

- https://shopify.dev/docs/api/app-home
- https://shopify.dev/docs/api/app-home/using-polaris-components